modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step05_common_geom.md

```diff
@@ -0,0 +1,206 @@
+         id=tutorial04-common_geo_hex_1
+         caption=Pin cell example input.
+         block=Mesh/hex_1
+
+## Assembly (Homogenized)
+
+Assemblies are one level higher than pins in the hierarchical lattice geometry structure. They consist of multiple pins laid out in a lattice, but sometimes, these pins are not explicitly represented. The [SimpleHexagonGenerator.md] mesh generator is appropriate for coarse, homogenized assembly approaches. Three modes are available for discretization: +TRI+ (6 triangles), +QUAD+ (2 quadrilaterals) and +HYBRID+ (6 triangles surrounded by layers of quadrilaterals). The +HYBRID+ method can be used when additional radial discretization is needed compared to the other two options. However, there is no additional azimuthal discretization available. [PolygonConcentricCircleMeshGenerator.md] should be used in that case. The hexagon will be discretized using the +TRI+ mode by default if no discretization scheme is specified.
+
+### Object
+
+- [SimpleHexagonGenerator.md]
+
+### Geometry Features
+
+- 2D Hexagonal unit pin cell with coarse mesh discretization
+
+### Notes
+
+- Three modes are available to discretize the hexagon:
+
+  - +TRI+: 6 triangles (default)
+  - +QUAD+: 2 quadrilateral
+  - +HYBRID+: 6 triangles + 6 \* layer number quadrilateral per assembly
+
+### Example
+
+!media tutorial04_meshing/base_ex_shg.png
+       id=tutorial04-base_ex_shg
+       caption=Example output of the TRI, QUAD, and HYBRID (one quadrilateral layer) mode, respectively, of [SimpleHexagonGenerator.md].
+       style=width:50%;display:block;margin-left:auto;margin-right:auto;
+
+!listing base_mesh_generators/common_geo.i
+         id=tutorial04-sim_hex.i
+         caption=Homogenized assembly example input.
+         block=Mesh/hex_simple
+
+## Assembly (with multiple heterogeneous pins)
+
+Assemblies with multiple heterogeneous pins are a very common reactor geometry component. The [PatternedHexMeshGenerator.md] creates an assembly by placing pre-defined pins into a user-defined lattice, and then applying optional background coolant and duct regions. Here we describe only the hexagonal assembly mesh generator. A Cartesian assembly mesh generator, [PatternedCartesianMeshGenerator.md], is also available.
+
+### Object
+
+- [PatternedHexMeshGenerator.md]
+- (Cartesian sibling -- [PatternedCartesianMeshGenerator.md])
+
+### Geometry Features
+
+- 2D Regular assembly with uniform pin pitch
+- Optional background and duct regions.
+
+### Notes
+
+- When generating an assembly mesh using [PatternedHexMeshGenerator.md], be sure to set [!param](/Mesh/PatternedHexMeshGenerator/generate_core_metadata) as `false`. This tells MOOSE that the constructed object is an assembly rather than a core (and consequently permits some operations downstream differently than if it were a core).
+- When generating an assembly mesh, [PolygonConcentricCircleMeshGenerator.md] objects which define the hexagonal unit pin cells are generally used as inputs.
```

Comment:
```suggestion
- When generating an assembly mesh, [PolygonConcentricCircleMeshGenerator.md] objects which define the hexagonal unit pin cells are generally used as inputs.
- The input pattern will be automatically rotated 90 degrees counterclockwise into a "vertex up" position unless otherwise specified with [!param](/Mesh/PatternedHexMeshGenerator/rotate_angle). This enables an assembly mesh to be patterned into a core directly (rows of assemblies should be oriented vertex up).
```